### ISSUE TITLE
Module on UNC Path Fails to Load

### ISSUE THREAD

**Issue by awickham10** [PowershellFrameworkCollective/psframework#227]
If the module is stored in a UNC path the module fails to load with the below error.

```
Exception calling "ReadAllText" with "1" argument(s): "The given path's format is not supported."
At \\server\share\psframework\0.10.27.135\PSFramework.psm1:57 char:9
```

If I remove the Resolve-Path it works correctly. Is Resolve-Path necessary here since you're not wildcarding any of the paths to Import-ModuleFile?

**Comment by FriedrichWeinmann** [comment]
Resolve-Path is in fact needed, since it is required for smooth Linux support (it corrects path separators depending on the current OS).
Fortunately, we can fix that with a small change: Have the command use the ProviderPath property on the output from `Resolve-Path` and things will be good.
Will also update the build mechanism on the nex version, so that it will by default no longer need to load any file at all.

**Comment by awickham10** [comment]
Good to know! Thanks for sharing that.

**Comment by FriedrichWeinmann** [comment]
Resolved in 0.10.28.144
